dlt/helpers/airflow_helper.py

```diff
@@ -179,7 +179,10 @@ def run(
 
         Args:
             pipeline (Pipeline): The pipeline to run
-            data (Any): The data to run the pipeline with
+            data (Any):
+                The data to run the pipeline with. If a non-resource
+                callable given, it's called before the load to get
```

Comment:
`before the load` is not helpful. what user should now is that it is called when the DAG is run, and not when the DAG is loaded - so just before actual pipeline run happens,.